lib/config/options/index.ts

```diff
@@ -722,6 +722,14 @@ const options: RenovateOptions[] = [
     subType: 'string',
     stage: 'repository',
   },
+  {
+    name: 'gitTimeout',
+    description:
+      'Configure the timeout with a number of milliseconds to wait for a git task',
+    type: 'integer',
```

Comment:
i think it needs to be `number` 🤔

Reply:
There isn't a `RenovateNumberOption` in config/types.ts there is only `RenovateIntegerOption`. You mean to add new type ?

Reply:
Oh, i see. There is a number array and an integer option.

@rarkins We should somehow unify those, maybe the `integer` option should be `number` or the other way around. WDYT?

Reply:
Yeah, for sure. Let's align with Json schema

Reply:
I think we can skip any change for this PR though

Reply:
@HonkingGoose Can you please create a corresponding issue if non exist?